src/scf/scf_callbacks.jl

```diff
@@ -40,28 +40,30 @@ function ScfDefaultCallback()
 
         # TODO We should really do this properly ... this is really messy
         if info.n_iter == 1
-            E_label = haskey(info.energies, "Entropy") ? "Free energy" : "Energy"
             label_magn = show_magn ? ("   Magnet", "   ------") : ("", "")
             label_damp = show_damp ? ("   α   ", "   ----") : ("", "")
             label_diag = show_diag ? ("   Diag", "   ----") : ("", "")
-            @printf "n     %-12s      Eₙ-Eₙ₋₁     ρout-ρin" E_label
+            @printf " n         Energy       log10(ΔE)   log10(Δρ)"
             println(label_magn[1], label_damp[1], label_diag[1])
-            @printf "---   ---------------   ---------   --------"
+            @printf "---   ---------------   ---------   ---------"
```

Comment:
I prefer the heading to be uniformly left-aligned rather than centred but not really.

Reply:
We can't left align the labels and right align the numbers

Reply:
Why not? Looks ok: https://github.com/JuliaMolSim/DFTK.jl/runs/5334962739?check_suite_focus=true#step:8:113